performance_regression_row_level_repair_test.py

```diff
@@ -297,9 +297,7 @@ def test_row_level_repair_large_partitions(self):
         clustering_row_size = 2048
         partition_count_per_node = partition_count // 10 // 3
         clustering_row_count_per_node = clustering_row_count // 100 * 3
-        scylla_bench_base_cmd = "scylla-bench -workload=sequential -mode=write -max-rate=300 " \
-                                "-replication-factor=3 -clustering-row-size={} -concurrency=10 -rows-per-request=30".format(
-                                    clustering_row_size)
+        scylla_bench_base_cmd = self.params.get('stress_cmd') + f" -clustering-row-size={clustering_row_size}"
```

Comment:
This test is  relying on Scylla Bench, assert if the stress command is different

Reply:
Added assertion